charts/newrelic-k8s-metrics-adapter/templates/configmap.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}
+  name: {{ template "common.naming.fullname" . }}
   labels:
-    {{- include "newrelic-k8s-metrics-adapter.labels" . | nindent 4 }}
+    {{- include "common.labels" . | nindent 4 }}
 data:
   config.yaml: |
     accountID: {{ .Values.config.accountID | required "config.accountID is required" }}
```

Comment:
Does make sense to have this as a global too?

Reply:
@gsanchezgavier, @paologallinaharbur ?

Reply:
As far as I know it is the only chart we own with that. StatsD has it and no idea about pixie.

Considering that we would need to implement it and then change chart we do not own, I would not say it is not really needed

Reply:
oks, we agreed to not add it as a global for the reasons exposed above.